### DISCUSSION TITLE
Dependency Warning on python3.11

### DISCUSSION THREAD

**Discussion by Akira-Kita** [simnibs/simnibs#296] [2. General]
Dear, Simnibs mannagers,

I tried to start simnibs on using python3.11, however this error massage like "/home/akira/SimNIBS-4.1/simnibs_env/lib/python3.9/site-packages/requests/__init__.py:102: RequestsDependencyWarning: urllib3 (1.26.18) or chardet (5.2.0)/charset_normalizer (2.0.4) doesn't match a supported version!" has appeared. And I cannot use 'import simnibs' on python3.11.
Please how to solve this problem?

Sincerly, Akira Kita

**Comment by TorgeW** [comment]
Python 3.11 is currently not supported. Installation might be possible but we are not directly supporting it yet.

**Reply by Akira-Kita** [reply]
Dear Torge W,

Thank you !

Sincerly, Akira Kita
